vlm/src/test/scala/geotrellis/contrib/vlm/gdal/GDALReaderSpec.scala

```diff
@@ -1,26 +0,0 @@
-package geotrellis.contrib.vlm.gdal
-
-import geotrellis.gdal._
-import geotrellis.raster.io.geotiff.reader.GeoTiffReader
-import geotrellis.raster.testkit._
-import geotrellis.spark.testkit._
-
-import org.scalatest._
-
-import java.io.File
-
-class GDALReaderSpec
-  extends FunSpec
-    with TestEnvironment
-    with RasterMatchers {
-
-  describe("GDALReaderSpec") {
-    it("should read full raster correct") {
-      val filePath = s"${new File("").getAbsolutePath()}/src/test/resources/img/aspect-tiled.tif"
-      val gdalTile = GDALReader(filePath).read().toArrayTile
```

Comment:
I would say that it won't be bad if it would be possible to have some test that compares the GeoTiff Reader and the GDAL bindings one.